addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1808,4 +1808,42 @@ options.registry.CoverProperties = options.Class.extend({
         this.$target[0].dataset.filterColor = activeFilterColor || '';
     },
 });
+
+/**
+ * Whether the section should be full-width (container-fluid) or use a classic container
+ */
+options.registry.section_stretch = options.Class.extend({
+    /**
+     * Only show this option if it changes the visual
+     * @override
+     */
+    start: function () {
+        const originalTarget = this.$target;
+
+        const unstretchedTarget = this.$target
+            .clone()
+            .removeClass('container-fluid')
+            .addClass('container');
+        // Elements have to be appended in the DOM to have a clientWidth
+        this.$target.replaceWith(unstretchedTarget);
+        const unstretchedWidth = unstretchedTarget[0].clientWidth;
+        
+        const stretchedTarget = this.$target
+            .clone()
+            .removeClass('container')
+            .addClass('container-fluid');
+        unstretchedTarget.replaceWith(stretchedTarget);
+        const stretchedWidth = stretchedTarget[0].clientWidth;
+
+        const sizeDifference = stretchedWidth/unstretchedWidth - 1;
```

Comment:
You don't need all of this, just the size of a container is enough. So something like:

```javascript
const $container = $('<div/>', {class: 'container'}).insertAfter(this.$target);
const sizeDifference = this.$target.parent().width() / $container.outerWidth() - 1;
$container.remove();
```